dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/vitals/VitalInfo.kt

```diff
@@ -0,0 +1,18 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.rum.internal.vitals
+
+internal data class VitalInfo(
+    val sampleCount: Int,
+    val minValue: Double,
+    val maxValue: Double,
+    val meanValue: Double
+) {
+    companion object {
+        val EMPTY = VitalInfo(0, Double.MAX_VALUE, -Double.MAX_VALUE, 0.0)
```

Comment:
No, this is on purpose. The update algorithm is : 

```
    val updatedInfo = VitalInfo(
        newSampleCount,
        min(newValue, vitalInfo.minValue),
        max(newValue, vitalInfo.maxValue),
        meanValue
    )
```
Because the `EMPTY.minValue` is set to `Double.MAX_VALUE`, the `updatedInfo.minValue` will always be `newValue` (because `newValue` will always be less or equal to `Double.MAX_VALUE`). Same goes for the `maxValue`. 
This avoid having to put `if (vitalInfo == EMPTY)` everywhere.

Reply:
ok, makes sense. I would suggest to add a comment here then.